src/model/Events.ts

```diff
@@ -10,9 +12,22 @@ export interface Event {
     responses: Response[];
 }
 
-export interface CompletedEvent {
-    event: Event;
-    response: Response;
-    feedback: Feedback;
-    reputation: Reputation[];
+export interface EndOfTurnSummary {
+    event: Event,
+    response: Response,
+    history: {
+        thisMonth: {
+            simulator: SimulatorMetrics[],
+            indicators: Indicators,
+            feedback: Feedback
+        },
+        lastMonth: {
+            simulator: SimulatorMetrics[],
+            indicators: Indicators
+        },
+        allTime: {
+            simulator: SimulatorMetrics[],
+            indicators: Indicators
+        },
+    }
```

Comment:
Old 'CompletedEvent' class you disliked, this is basically just info the front-end needs to show players